test/tests/functional/pbs_qstat_formats.py

```diff
@@ -177,7 +177,7 @@ def get_qstat_attribs(self, obj_type):
             #        "PBS_O_QUEUE":"workq",
             #    },
 
-            if ',' in val:
+            if ',' in val and key != 'managers':
```

Comment:
what about 'operators'? Since this seems to be specific for the Variable_List attribute, I think you should just match that exactly (if key == ATTR_v) instead of matching a comma